packages/ui-components/src/lib/form/Form.tsx

```diff
@@ -39,7 +39,6 @@ export interface FormProps<T> extends Omit<FormikConfig<T>, 'onSubmit'> {
   unsavedChangesConfig?: UnsavedChangesConfig;
   withoutFormElement?: boolean;
   formConfig?: FormConfig;
-  children: ((formikProps: FormikProps<T>) => ReactElement) | ReactElement | ReactElement[];
```

Comment:
I didn't see the need for this, since `FormikConfig` (which `FormProps` extends) already includes the valid type for `children`.